### DISCUSSION TITLE
unable to run Barrier on Apple Silicon macs

### DISCUSSION THREAD

**Discussion by ihubgit** [debauchee/barrier#2037] [Q&A]
Hi i saw this working flawlessly a while back between windows and Apple Intel machine at a friend's place, but i have been absolutely unsuccessful to make barrier work between my M1 max mac studio and M2 macbook air.

Both have their own ip address and they're connected in the local LAN over an ethernet cable.
File sharing works with nitroshare f.ex. , but when configuring the mac studio as server and the macbook air as client, disabling SSL, it just says the server/client is starting, but never ends up runnig.

Why?

**Comment by DeepYogurt** [comment]
If you set logging level to something more detailed than Info (like debug or debug1 or debug2) do you get any hints as to what it's complaining about? You set this logging level in the Settings pane via the menu Barrier then "Change Settings".

So, Barrier on both the client and sever say they aren't running or just one of them (if just one, which one)??

And you're running Barrier 2.4.0 on both? and both mac systems are running which macOS version?

I'm running my Barrier on three macs - one an M2 Pro Mac Mini on Sonoma (have run it on Ventura and Sequoia as well) running version 2.4, and an iMac running Mojave running Barrier version 2.4 and an old TrashCan MacPro running High Sierra using Barrier 2.2.3 - all work fine for the most part. I do have the SSL tuned on.

Also, you've got Barrier enabled to "control your computer" in the System Settings/Privacy & Security/Accessibility pane also, right?

The only problem at the moment is on the High Sierra 2.3.3 client version of Barrier, there is sometimes quite a time lag between when I try to connect from iMac Barrier server to this system before it actually connects - about 20-30 seconds typically. There is no time lag between the Mojave iMac Barrier server and the Ventura/Sonoma/Sequoia Barrier client.

Good luck...

-bob
